radish34/contracts/__tests__/OrgRegistry.test.js

```diff
@@ -59,12 +59,45 @@ test('Should be able to register an org', async () => {
     utils.hexlify('0x0471099dd873dacf9570f147b9e07ebd671e05bfa63912ee623a800ede8a294f7f60a13fadf1b53d681294cc9b9ff0a4abdf47338ff72d3c34c95cdc9328bd0128'),
     utils.hexlify('0x0471099dd873dacf9570f147b9e07ebd671e05bfa63912ee623a800ede8a294f7f60a13fadf1b53d681294cc9b9ff0a4abdf47338ff72d3c34c95cdc9328bd0128')
   );
+  const orgCountBefore = await orgRegistry.getOrgCount();
+  expect(orgCountBefore.toNumber()).toBe(1);
+});
+
+test('Should not be able to register an org from non-owner', async () => {
+  const nonOwner = await getSigner(accounts[4]);
```

Comment:
you could create a random wallet using ethers

Reply:
Addressed